.github/workflows/test_fast.yml

```
@@ -39,24 +41,26 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Configure Python
-        uses: actions/setup-python@v5
+      - name: Install System Dependencies
+        uses: mamba-org/setup-micromamba@v2
         with:
-          python-version: ${{ matrix.python-version }}
-
-      - name: Apt-Get Install
-        if: startsWith(matrix.os, 'ubuntu')
+          generate-run-shell: true
+          cache-environment: true
+          post-cleanup: 'all'
+          environment-name: cylc-fast-test
+          create-args: >-
+            python=${{ matrix.python-version }}
+            pip
+            bash
+            sqlite
```

I've noticed the SVN unit tests are being skipped. Which is strange because we didn't install SVN before. Is it possible to include SVN here?

---

Good spotting.

This suggests that if subversion is installed, the `svn` executable is no longer in the `$PATH`.

I'm not sure if this is a consequence of the switch to micromamba (doesn't make sense) or a consequence of dropping `apt-get` installation (doesn't make sense).

---

We could just do this:

```suggestion
            sqlite
            subversion
```

---

> Ah wait, it's because it was installed on Ubuntu 22 but not latest. Is it possible to include SVN here?

I have just discovered this myself.

Let's try installing it from conda-forge on the newer images.

---

Done, will see if it works...